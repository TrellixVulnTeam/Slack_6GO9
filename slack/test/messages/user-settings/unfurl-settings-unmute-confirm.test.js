const { UnfurlSettingsUnmuteConfirm } = require('../../../lib/messages/user-settings');

describe('UnfurlSettingsUnmuteConfirm rendering', () => {
  test('works', async () => {
    expect(new UnfurlSettingsUnmuteConfirm().getAttachment()).toMatchSnapshot();
  });
});
