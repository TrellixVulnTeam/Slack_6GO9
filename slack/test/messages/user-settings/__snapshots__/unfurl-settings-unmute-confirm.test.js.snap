// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnfurlSettingsUnmuteConfirm rendering works 1`] = `
Object {
  "color": "#24292f",
  "text": ":white_check_mark: Prompts to show a rich preview have been unmuted",
}
`;
